frontend/src/App.tsx

```diff
@@ -30,25 +32,64 @@ function App() {
     <Layout>
       <>
         <section className='flex text-main-white'>
-          {data.map((post: Post) => (
-            <article key={post.id} className=' w-1/2'>
-              <img
-                src='https://duet-cdn.vox-cdn.com/thumbor/0x0:6592x4399/1200x800/filters:focal(3296x2200:3297x2201):format(webp)/cdn.vox-cdn.com/uploads/chorus_asset/file/23893495/1238141654.jpg'
-                alt='rocket'
-                className='w-full h-auto'
-              />
-              <h1 className=' text-5xl font-extrabold mb-3 -mt-4 relative hover:text-main-red'>
-                {post.title}
-              </h1>
-              <p className='text-base font-extralight text-stone-200 '>
-                {post.body.slice(0, 200)}...
-              </p>
-              <div className='flex mt-2 text-sm font-light text-stone-400'>
-                <p className='mr-6 text-main-red'>{post.author}</p>
-                <p className=''>{post.date.split('T')[0]}</p>
-              </div>
-            </article>
-          ))}
+          {isSuccess &&
+            data
+              .reverse()
```

Comment:
An error is thrown here when we click on a post on the home page and go back. 
The following error is raised:
```javascript
Unhandled Thrown Error!
data.reverse is not a function
TypeError: data.reverse is not a function
    at App (http://localhost:3000/src/App.tsx:60:38)
    at renderWithHooks (http://localhost:3000/node_modules/.vite/deps/chunk-XY35XOXX.js?v=c36de336:11761:26)
    at mountIndeterminateComponent (http://localhost:3000/node_modules/.vite/deps/chunk-XY35XOXX.js?v=c36de336:14490:21)
    at beginWork (http://localhost:3000/node_modules/.vite/deps/chunk-XY35XOXX.js?v=c36de336:15445:22)
    at beginWork$1 (http://localhost:3000/node_modules/.vite/deps/chunk-XY35XOXX.js?v=c36de336:19246:22)
    at performUnitOfWork (http://localhost:3000/node_modules/.vite/deps/chunk-XY35XOXX.js?v=c36de336:18691:20)
    at workLoopSync (http://localhost:3000/node_modules/.vite/deps/chunk-XY35XOXX.js?v=c36de336:18630:13)
    at renderRootSync (http://localhost:3000/node_modules/.vite/deps/chunk-XY35XOXX.js?v=c36de336:18609:15)
    at recoverFromConcurrentError (http://localhost:3000/node_modules/.vite/deps/chunk-XY35XOXX.js?v=c36de336:18231:28)
    at performSyncWorkOnRoot (http://localhost:3000/node_modules/.vite/deps/chunk-XY35XOXX.js?v=c36de336:18373:28)
```